frontend/packages/console-app/src/components/volume-snapshot/volume-snapshot-class.tsx

```diff
@@ -12,14 +12,17 @@ import {
 } from '@console/internal/components/factory';
 import { VolumeSnapshotClassModel } from '@console/internal/models';
 import { getBadgeFromType, BadgeType } from '@console/shared';
+import { getAnnotation } from '@console/kubevirt-plugin/src/selectors/selectors';
```

Comment:
do not import from kubevirt plugin

Reply:
sorry for the delayed update. i've fixed this :)

Reply:
sorry for delayed update. i've removed kubevirt plugin